presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -198,8 +198,18 @@ public LocalQueryRunner(Session defaultSession)
         this(defaultSession, false);
     }
 
+    public LocalQueryRunner(Session defaultSession, FeaturesConfig featuresConfig)
+    {
+        this(defaultSession, featuresConfig, false);
+    }
+
     private LocalQueryRunner(Session defaultSession, boolean withInitialTransaction)
     {
+        this(defaultSession, new FeaturesConfig().setExperimentalSyntaxEnabled(true), withInitialTransaction);
+    }
+
+    private LocalQueryRunner(Session defaultSession, FeaturesConfig featuresConfig, boolean withInitialTransaction)
```

Comment:
@martint Does adding `FeatureConfig` to constructor of `LocalQueryRunner` look good to you?